summingbird-storm/src/main/scala/com/twitter/summingbird/storm/collector/KeyValueOutputCollector.scala

```diff
@@ -0,0 +1,30 @@
+package com.twitter.summingbird.storm.collector
+
+import backtype.storm.spout.SpoutOutputCollector
+import java.util.{ List => JList }
+
+/**
+ * Created by pnaramsetti on 7/19/16.
+ *
+ * The KeyValueOutputCollector is used to unwrap the Value object when passed on by the Spout.
+ */
+class KeyValueOutputCollector(self: SpoutOutputCollector, func: JList[AnyRef] => JList[AnyRef]) extends SpoutOutputCollector(null) {
```

Comment:
what has this to do with `KeyValue`? Looks like a generic `TransformingOutputCollector` to me. It can totally generally transform right?